### PR TITLE
Tumor size over time

### LINKED ISSUE #177: Add new algorithm for Tumor Size Over Time
This is based on a SAS program that IMS maintains.

I already provided the program.

To add a new algorithm, you will need to create a "tumorsizeovertime" package, and a TumorSizeOverTime algorithm that extends the Algorithm interface. You will need to register it in Algorithms.  And finally, you will need to add a new (non-standard) field in Algorithms for that computed field.  You can assign the next NAACCR number available for the non-standard fields.

I *think* that's it. There are plenty of examples.

Let me know if you have any questions.

### PR DESCRIPTION
**Pull request by bekeles**
closes #177

### CODE REVIEW

File: src/main/java/com/imsweb/algorithms/Algorithms.java [hunk 1]
```diff
@@ -188,6 +190,12 @@ public final class Algorithms {
     public static final String FIELD_CDC_SVI_2018 = "cdcSVI2018";
     public static final String FIELD_DAYS_TO_TREATMENT = "daysToTreatment";
 
+    public static final String FIELD_TUMOR_SIZE = "csTumorSize";
```
Comment (depryf) [inline src/main/java/com/imsweb/algorithms/Algorithms.java:193]: Aren't these two standard fields? I know it's a bit silly, but there is a group of standard fields and another one after that with the non-standard. Could you please move these to the standard block?

File: src/main/java/com/imsweb/algorithms/tumorsizeovertime/TumorSizeOverTimeUtils.java [hunk 2]
```diff
@@ -0,0 +1,228 @@
...
+import org.apache.commons.lang3.Range;
+import org.apache.commons.lang3.StringUtils;
+import org.apache.commons.lang3.math.NumberUtils;
+
+import com.opencsv.CSVReader;
+import com.opencsv.exceptions.CsvException;
+
+public final class TumorSizeOverTimeUtils {
+
+    public static final String ALG_NAME = "Tumor Size Over Time";
```
Comment (depryf) [inline src/main/java/com/imsweb/algorithms/tumorsizeovertime/TumorSizeOverTimeUtils.java:26]: By convention, we put the owner in front of the name. Can you please put "SEER" in front of the current name?

**Review by depryf — CHANGES_REQUESTED** [review]

File: src/main/java/com/imsweb/algorithms/Algorithms.java (hunk 1 above)
Comment (bekeles) [reply src/main/java/com/imsweb/algorithms/Algorithms.java:193]: Sure. I didn't notice that.

File: src/main/java/com/imsweb/algorithms/tumorsizeovertime/TumorSizeOverTimeUtils.java (hunk 2 above)
Comment (bekeles) [reply src/main/java/com/imsweb/algorithms/tumorsizeovertime/TumorSizeOverTimeUtils.java:26]: Sure!

**Comment by bekeles** [comment]
@depryf did this slip away from your list?

**Comment by depryf** [comment]
It sure did! Looks good, merging now.

**Review by depryf — APPROVED** [review]

**Pull request merged**